waterlog/src/Globals.js

```diff
@@ -1,3 +1,3 @@
 export const Globals = {
-	API_URL: 'https://api.iot.retrotest.co.za'
+	API_URL: 'https://localhost:44382/'
```

Comment:
before you create a pr. Review and make sure you changed the api_url back and is not localhost

Reply:
Thanks for the reminder. I fixed it.